releasenotes/notes/decompose-controlflow-7a7e38d402aed260.yaml

```
@@ -0,0 +1,11 @@
+---
+fixes:
+  - |
+    Fixed a bug where calling :meth:`.QuantumCircuit.decompose` on an instruction
+    that had no definition and in a ``c_if`` block would raise an error.
```

Maybe this wording makes it a bit cleare?

```suggestion
    that had no definition inside a ``c_if`` block would raise an error.
```